next.config.js

```diff
@@ -12,4 +12,4 @@ const nextConfig = {
   },
 };
 
-module.exports = nextConfig;
+module.exports = nextConfig;
```

Comment:
when you see the red sign. be sure to add a new line at the end of every javascript or css file you add to your project or repo.